### DISCUSSION TITLE
Prioritizing New Posts in Blog Searches

### DISCUSSION THREAD

**Discussion by caklein** [Pagefind/pagefind#494] [General]
Hello! I've been using Pagefind for my blog and I'm quite impressed with its capabilities. However, I'm facing a challenge: I want newer posts on similar topics to appear before older ones in search results. We tried manipulating the search weight, assigning a weight of 10 to the most recent posts, 9 to posts between 24 hours and one week old, and so on, but this approach didn't yield the desired outcome.

Is there a way to achieve this prioritization of newer content over older content using Pagefind's default UI? Any guidance or tips on how to effectively sort search results by recency would be greatly appreciated.

**Comment by bglw** [comment]
Hmm, Pagefind does support sorting, but only as a replacement for ranking rather than in addition to.

It's an interesting idea to merge the two though! And I think it would be achievable with the current setup, so I'll mull that over.

The weighting, if applied to the body element as a whole, _would_ provide something close to this though. With the downside that Pagefind would no longer automatically weight headings higher or lower, so not the best solution. 

I'll jump back here if I think of anything else (or make progress on merging the relevancy and sorting!)

**Comment by caklein** [comment]
Hello,

I hope this message finds you well. I've been trying to organize my results by implementing a logic to give priority to newer content, but unfortunately, I haven't achieved the desired success.

I was wondering if there is a possibility of implementing a feature that allows specifying the default sort order, disregarding the weights. Such a addition would be immensely helpful in resolving the challenge I'm facing.

Thank you very much for your consideration and assistance.

Best regards,

**Reply by bglw** [reply]
Specifying a sort order will disregard the weights, so this should be available in the current state. Let me know if that isn't what you're seeing!
